src/main/java/edu/usf/cutr/grha/parsing/CsvBeanFileParser.java

```diff
@@ -5,9 +5,12 @@
 import com.univocity.parsers.csv.CsvParser;
 import com.univocity.parsers.csv.CsvParserSettings;
 import edu.usf.cutr.grha.ProcessorMain;
+import edu.usf.cutr.grha.model.ChicagoData;
 import edu.usf.cutr.grha.model.LocationFixData;
+import edu.usf.cutr.grha.utils.ChicagoDataUtils;
 import edu.usf.cutr.grha.utils.LocationFixDataUtils;
 
+import java.io.Reader;
 import java.util.List;
 
 public class CsvBeanFileParser extends ProcessorMain {
```

Comment:
Is there a reason why this `extends ProcessorMain`? I would think it would be it's own class, not extending anything.

I'd also recommend splitting each file type parsing into it's own parser. So you could have `ChicagoDataParser`, `GPSTestParser`, etc.

Reply:
I was extending the `ProcessorMain` to make use of the `getReader()` declared in the class, looked neater than calling `ProcessorMain.getReader()`.